src/test/java/seedu/address/logic/commands/edit/EditUserDescriptorTest.java

```diff
@@ -0,0 +1,93 @@
+package seedu.address.logic.commands.edit;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_CAP_BOB;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_DESCRIPTION_BOB;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_DP_BOB;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_EMAIL_BOB;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_FROM_BOB;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_GITHUB_BOB;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_MAJOR_BOB;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_NAME_BOB;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_PHONE_BOB;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_TO_BOB;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_UNIVERSITY_BOB;
+import static seedu.address.testutil.TypicalEditUserDescriptor.AMY_DESC;
+import static seedu.address.testutil.TypicalEditUserDescriptor.BOB_DESC;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.testutil.EditUserDescriptorBuilder;
+
+public class EditUserDescriptorTest {
+    @Test
+    public void isAnyFieldMissing_noFieldEdited_returnsTrue() {
```

Comment:
should this be `isAnyFieldEdited_noFieldEdited_returnsFalse()` ?

Reply:
Yes